tests/code_fellows/test_linked_list.py

```diff
@@ -1 +1,23 @@
-# https://codefellows.github.io/sea-python-401d6/assignments/linked_list.html
\ No newline at end of file
+# https://codefellows.github.io/sea-python-401d6/assignments/linked_list.html
+
+from ...code_fellows.linked_list import LinkedList, Node
+
+
+def test_Linked_List():
+    assert LinkedList(["head"]).head.value == "head"
+    assert (
+        LinkedList([3, "three"]).head.value == "head"
+        and LinkedList([3, "three"]).head.next.value == 3
+    )
+
+
+def test_Linked_List_push():
+    test_ll = LinkedList([999, 55, 1])
```

Comment:
This is in a test so the `test_` prefix on `test_ll` is a bit redundant. If it's to avoid linting complaints, you could make it simple_ll or linked_list